client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/style.scss

```diff
@@ -296,6 +305,7 @@ $font-family: "SF Pro Text", $sans;
 		display: flex;
 		flex-direction: column;
 		flex-grow: 1;
+		overflow-x: visible;
```

Comment:
This is required because `.screen-container__body--align-sides` uses negative margins. We can update the sidebar width and the screen paddings later. I'm using negative margins as a quick fix.